src/backend/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.7-alpine AS base
```

Comment:
use official python image, do a 3 stage build:
* base - pipenv, env vars, workdir, and runtime deps
* build - compilers, headers, git, things needed to build python deps
* final - base + built deps only